ci/build.yml

```diff
@@ -409,11 +411,9 @@ jobs:
   condition: always()
   dependsOn:
     - Linux
-    - macOS
```

Comment:
I don't think you need to make any changes here, the `collect_build_data` job should be OK with some of its dependencies not running.

Reply:
See [prs](https://github.com/digital-asset/daml/blob/main/ci/prs.yml#L76-L78): dependencies need to exist, but not running is OK.

Reply:
ff2a948d333cf8d55080228f215f4b1e32886f89 thanks!